nussl/core/constants.py

```diff
@@ -54,9 +54,8 @@
 """list(str): list of all available windows in *nussl*
 """
 
-NUMPY_JSON_KEY = (
-    "py/numpy.ndarray"
-)  #: (str): key used when turning numpy arrays into json
+NUMPY_JSON_KEY = "py/numpy.ndarray"
```

Comment:
Super minor but could we put the comment above the constant? I think the way it is currently it's a little less clear which comments refer to which lines of code.

Reply:
I don't recall the details 100%, but I remember that I put the comments after the constant because that's how the sphinx formatting worked to pick up single line comments. We might be able to finagle something better though...